packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -468,6 +469,19 @@ export function injectIntoGlobalHook(globalObject: any): void {
       return id;
     };
 
+    // Do the same for any already injected roots.
```

Comment:
Note this is almost copy paste from the code above (462–468). I could extract a function but meh.